src/context/reducers/groupChat.js

```diff
@@ -2,19 +2,19 @@ import {SET_ASSET_GROUP_CHAT, SET_PARTICIPANTS_GROUP_CHAT} from '../Types';
 
 const groupChatState = {
   asset: [],
-  participants: {},
+  participants: []
 };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The initial state of `participants` has been changed from an object to an array. This could potentially cause issues if the rest of your code is expecting `participants` to be an object. Please ensure that all references to `participants` in your code are updated to handle an array instead of an object.

```diff
-  participants: {},
+  participants: [],
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->